### PR TITLE
adding geometry folder for respective cat of nodes

### PR DESCRIPTION
**Pull request by og76**
#430

### CODE REVIEW

**Comment by JacquesLucke** [comment]
ok, this is a good idea. Can you move the files around and make a pull request?

**Comment by og76** [comment]
this is it #430

File: nodes/geometry/project_point_on_plane.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        if isLinked["projection"]:
+            yield "int = mathutils.geometry.intersect_line_plane(point, point + plane_no, plane_co, plane_no, False)"
+            yield "projection = mathutils.Vector((0, 0, 0)) if int is None else int"
+        if isLinked["distance"]: yield "distance = mathutils.geometry.distance_point_to_plane(point, plane_co, plane_no)"
+ 
+    def getUsedModules(self):
+        return ["mathutils"]
+
+def getPlane():
+    return "planePoint, planeNormal if planeNormal.length_squared != 0 else mathutils.Vector((0, 0, 1))"
```
Comment (JacquesLucke) [inline nodes/geometry/project_point_on_plane.py]: have you tested this?
Looks in the case that the length is 0, there will only be one value for two variables

Comment (og76) [inline nodes/geometry/project_point_on_plane.py]: yep, the if only refers to normal, I'd have to put it in () to refer to both
pointplaneworks

**Comment by JacquesLucke** [comment]
Looks good so far except for the one little thing I mentioned.

Also maybe the user should be able to search for "Distance Point to Line" or so? what do you think?
Projection is not what the normal user thinks of when it comes to measuring distances.

**Comment by og76** [comment]
_I've worked with these for a while now, so they are ok._

They are projections, as in geometry. The distance is not the main deal, imho. Like the kdtree giving also the distance, but not the main deal

At most, **closest point to line/plane** is more readable. Distance is not enough ...hm. it gives you the point .. in the line it also gives 0-1 factor for inside line ..

**Comment by JacquesLucke** [comment]
Yes I agree that the distance is not the main deal of these nodes but it is their main use for most users (at least I think so). So I still think users should find these nodes with multiple search tags. Similiar to the Time Info node which you can also find by typing "frame" in the search.
(we don't need the node twice in the menu)

**Comment by og76** [comment]
should I also rename Distance to plane to **Signed Distance to plane** ? 
Cause it is not obvious, and in Line it's not

**Comment by JacquesLucke** [comment]
Hmmmm okay.. yes, maybe it's better to have 'signed' in the name.

**Pull request merged**